### DISCUSSION TITLE
Best way to track an "incoming transaction" from a "Fixed Term Deposit"

### DISCUSSION THREAD

**Discussion by JcMinarro** [firefly-iii/firefly-iii#8532] [Firefly III]
First of all, a big shout-out to JC5 for this project.

I would like to know the "firefly way" to track an incoming transaction from a fixed-term deposit.
Imagine the following scenario.
I open a fixed-term deposit that will return me the deposit amount + 3% in 1 year.
For the deposit, I create an asset account, then I do a transfer from my bank account to the deposit account. Let's say the transfer is 1000€.
Then, after 1 year, my bank returns my the deposit I made plus the interest. It is a transaction from the deposit account to my bank account for 1030€.
Here is where I have the question, how should I track this transaction?
1-. I don't want it to be an "incoming transactions" of 1030€ because on that case I would have a wrong incoming amount on any report I do (1000€ was transferred between my accounts, not incoming, I only got 30€ extra) 
2-. Looking into [Split Transaction Docs](https://docs.firefly-iii.org/explanation/financial-concepts/transactions/#what-are-split-transactions) it looks like I can't split a transfer into one transfer and one deposit
3-. Alternatively, I can duplicate the transaction, edit the amounts of them (1000€ and 30€), and convert one deposit transaction into a transfer to avoid "artificial incoming"

Am I losing anything?

**Comment by JC5** [comment]
Could it not simply be another asset account, where you transfer the money to? Then, a deposit is made by your bank for 30.

**Reply by JcMinarro** [reply]
Currently I have 2 assets accounts (Bank and Deposit) and 1 revenue account (Bank-Revenue).
The problem is that a single transaction is a transfer and an incoming at the same time, but Firefly III doesn't allow it.
Let's me show you an example of the transactions:
A - 01/01/2023 -> 1000€ from "Bank" to "Deposit"
B - 31/12/2023 -> 1030€ from "Deposit + Bank-Revenue" to "Bank"

The problem is that only one transaction for 1030€ is recorded and it can't be split into "transfer and incoming".
So, the only solution I found is to duplicate transaction B and edit the amounts like:
B1 - 31/12/2023 -> 1000€ from "Deposit" to "Bank"
B2 - 31/12/2023 -> 30€ from "Bank-Revenue" to "Bank"

Is it the way to go? Or there is any alternative?

**Reply by JC5** [reply]
That's the way to go, yes.
